en/services/timesync_v1.md

```diff
@@ -0,0 +1,45 @@
+# Time Synchronization Protocol v1
+
+This protocol is used to estimate the time offsets between system clocks on two MAVLink systems, in a two-system MAVLink network.
+This is useful because it allows ....
```

Comment:
What are the use cases? I understand this gives you the difference in system clock, but when might you need to know that?

Reply:
@peterbarker So why do you need this?

Reply:
It's generally nice for people who are learning about a system to understand what it is for/why they should care.